tvb/simulator/plot/plotter.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+from tvb.simulator.plot.base_plotter import BasePlotter
+from tvb.simulator.plot.head_plotter import HeadPlotter
+from tvb.simulator.plot.time_series_plotter import TimeSeriesPlotter
+
+
+class Plotter(object):
+
+    def __init__(self, config=None):
+        self.config = config
+
+    def tvb_plot(self, plot_fun_name, *args, **kwargs):
+        return BasePlotter(self.config).\
+            plot(plot_fun_name, *args, **kwargs)
+
+    def plot_head(self, head):
```

Comment:
The same Head class is indirectly referred here also

Reply:
The proposed solution can be seen in a comment above.